templates/rhdm73-prod-immutable-kieserver-amq.yaml

```diff
@@ -0,0 +1,1005 @@
+---
```

Comment:
For the master branch, this file name needs to have an rhdm74 prefix, not rhdm73.

Reply:
Also all "73 and 7.3" references in the file need to get updated as well.

Reply:
ok, old PR :) , done before the version update